cloud/blockstore/vhost-server/request_aio_ut.cpp

```diff
@@ -495,7 +497,105 @@ Y_UNIT_TEST_SUITE(TAioRequestTest)
 
     Y_UNIT_TEST_F(ShouldPrepareCompoundIOForSplitDevices, TTestBackend)
     {
-        InitSplitDevices(5, 1_MB);
+        InitSplitDevices(5, 1_MB, 512_B);
```

Comment:
let's extract the test into function `CheckPrepareOfCompoundIOForSplitDevices(ui32 blockSize)`
and use it in tests
`ShouldPrepareIOForSplitDevices512` 
`ShouldPrepareIOForSplitDevices4KB`